recipe/meta.yaml

```diff
@@ -12,13 +12,14 @@ source:
 build:
   noarch: python
```

Comment:
```suggestion
```

We have a policy to not add noarch packages. https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Recipe_standards.md#the-build-section